### DISCUSSION TITLE
any tooltip or something can indicate whether I am in fullscreen mode now?

### DISCUSSION THREAD

**Discussion by OriLiMu** [zellij-org/zellij#3339] [Q&A]
When I use fullscreen pane mode. There is no tooltip or indication of it. Sometime I may forgot that. So any way to make a indication?

It may some words in the title (fullscreen) something

**Comment by cristiand391** [comment]
the default layout includes the `status-bar` at the bottom which indicates when in fullscreen:

![image](https://github.com/zellij-org/zellij/assets/6853656/3786024e-e43d-4aa6-b2be-a0c5d659eec2)

Other than that neither `tab-bar` or `compact-bar` built-in plugins does this, see:
https://github.com/zellij-org/zellij/issues?q=is%3Aissue+fullscreen+indicator

I maintain a fork of `compact-bar` which has support for this, see the GIF here:
https://github.com/zellij-org/zellij/issues/3074#issuecomment-1901289975

There's no readme yet but the wasm build is available:
https://github.com/cristiand391/zj-status-bar/releases

**Reply by imsnif** [reply]
If you'd like to add some docs and then add it to [awesome zellij](https://github.com/zellij-org/awesome-zellij) that'd be cool!

**Reply by OriLiMu** [reply]
> the default layout includes the `status-bar` at the bottom which indicates when in fullscreen:
> 
> ![image](https://private-user-images.githubusercontent.com/6853656/329436692-3786024e-e43d-4aa6-b2be-a0c5d659eec2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcyNDQwOTEsIm5iZiI6MTcxNzI0Mzc5MSwicGF0aCI6Ii82ODUzNjU2LzMyOTQzNjY5Mi0zNzg2MDI0ZS1lNDNkLTRhYTYtYjJiZS1hMGM1ZDY1OWVlYzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDFUMTIwOTUxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGIyMTQxODY1MTc2NDdkNDkyMjM0YzE1ZmUwMjlkOTFiZWE5MjIxNzU0ZWZmNDM0ODAyZThkM2FhNDY2M2ZjMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.1cDmXgPTcC7E2e8Jyhm1FYrzPlxS6pEYQmi8Nl-_yWM)
> 
> Other than that neither `tab-bar` or `compact-bar` built-in plugins does this, see: https://github.com/zellij-org/zellij/issues?q=is%3Aissue+fullscreen+indicator
> 
> I maintain a fork of `compact-bar` which has support for this, see the GIF here: [#3074 (comment)](https://github.com/zellij-org/zellij/issues/3074#issuecomment-1901289975)
> 
> There's no readme yet but the wasm build is available: https://github.com/cristiand391/zj-status-bar/releases

Any way to add the `fullscreen tip` to the tab bar? I don't like the status bar take to much room for screen

**Reply by cristiand391** [reply]
> Any way to add the fullscreen tip to the tab bar? I don't like the status bar take to much room for screen

nope. That's one of the main reasons (along with tab indexes) I forked it (updated readme with some screenshots/instructions from my last comment 😉 ).
